CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Documentation on the release process ([#241](https://github.com/stac-utils/stactools/pull/241))
 - `stac lint` using [stac-check](https://github.com/philvarner/stac-check) ([#254](https://github.com/stac-utils/stactools/pull/254))
 - `stac info` now has option `-s` or `--skip_items` to skip counting and printing catalog item info ([#14](https://github.com/stac-utils/stactools/issues/14))
+- updated `livehtml` target for documentation build to work with latest `sphinx-autobuild`
```

Comment:
Can you please update this entry w/ a link to the pull request (ala line 13)? Can you also make the same change for line 14, to reference #260 instead of #14? I missed that last night. Thanks.

Reply:
Done, thanks!